components/supervisor/frontend/src/index.ts

```diff
@@ -11,6 +11,13 @@
 import { IDEMetricsServiceClient, MetricsName } from "./ide/ide-metrics-service-client";
 IDEMetricsServiceClient.addCounter(MetricsName.SupervisorFrontendClientTotal).catch(() => {})
 
+try {
+    if (!sessionStorage.getItem('gitpodSessionReportedLoad')) {
```

Comment:
Only ever report the session as "loaded" once per browser session  => window reloads will **not** result in duplicate reports

Reply:
> A page session lasts as long as the tab or the browser is open, and survives over page reloads and restores.

From https://developer.mozilla.org/en-US/docs/Web/API/Window/sessionStorage We want to track one each page load.